docs/mutect2/strand-artifact-filter.tex

```diff
@@ -0,0 +1,85 @@
+\documentclass[a4paper]{article}
+
+%% Language and font encodings
+\usepackage[english]{babel}
+\usepackage[utf8x]{inputenc}
+\usepackage[T1]{fontenc}
+
+%% Sets page since and margins
+\usepackage[a4paper,top=3cm,bottom=2cm,left=3cm,right=3cm,marginparwidth=1.75cm]{geometry}
+
+%% Useful packages
+\usepackage{amsmath}
+\usepackage{graphicx}
+\usepackage[colorinlistoftodos]{todonotes}
+\usepackage[colorlinks=true, allcolors=blue]{hyperref}
+
+\title{Mutect2 Strand Artifact Filter}
+\author{Takuto Sato}
+
+\begin{document}
+\maketitle
+
+\section{Introduction}
+
+
+Strand artifact filter captures false positives that arise from . This model is a build 
+
+\section{The Probabilistic Model}
+\begin{figure}
+\centering
+\includegraphics[width=0.3\textwidth]{strand_artifact_pgm.png}
+\caption{\label{fig:frog}The probabilistic graphical model}
+\end{figure}
+
+\begin{itemize}
+	\item $z \sim \text{Multinomial}(\pi)$ is a latent variable, encoded as a 3-dimensional vector of the probability of strand artifact in forward reads (index 0), reverse reads (1), or neither (2). For instance, $z = [0.9, 0.03, 0.07]$ means that we believe that any given \textit{forward} ref read is subject to strand artifact with probability 0.9.
```

Comment:
I should say Cateogorical, not Multinomial, to describe the distribution of z, right?